tests/chaos/oadp/test_oadp.py

```
@@ -68,3 +73,265 @@ def test_drain_vm_node_during_backup(
+    expected_status,
+):
+    """
+    This test verifies OADP restore resilience under control-plane disruptions.
+
+    High-level flow:
+    1. Create a healthy VM and persist data inside the guest.
+    2. Take a successful OADP backup from a stable cluster state.
+    3. Explicitly stop and delete the original VM to ensure the restore path
+       recreates the VM from backup artifacts.
+    4. Start a background process that continuously deletes critical OADP-related
+       pods (e.g. controller-manager, node-agent，velero，minio) while the restore is in progress.
```

Ruff (0.14.10)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)